environments/includes/elasticsearch.base.yml

```diff
@@ -3,17 +3,21 @@ services:
   elasticsearch:
     hostname: "${WARDEN_ENV_NAME}-elasticsearch"
     image: docker.io/wardenenv/elasticsearch:${ELASTICSEARCH_VERSION:-7.8}
-    labels:
-      - traefik.enable=true
```

Comment:
I would keep it both in load balancer.
`elasticsearch.${TRAEFIK_DOMAIN}` and `elastichq.${TRAEFIK_DOMAIN}`